### PR TITLE
Updated git-annex documentation

### PR DESCRIPTION
**Pull request by kousu**

### CODE REVIEW

File: CONTRIBUTING.md
Comment (jcohenadad) [inline CONTRIBUTING.md]:
```suggestion
3. Start a branch:
```

Comment (jcohenadad) [reply CONTRIBUTING.md]: (pushing all the numbers below 😉 )

**Review by jcohenadad — COMMENTED** [review]
awesome! just the minor issue with the numbering. thanks 🙏

**Review by jcohenadad — APPROVED** [review]

File: CONTRIBUTING.md
Comment (kousu) [reply CONTRIBUTING.md]: True. I'll fix that.

**Comment by kousu** [comment]
@jcohenadad I added the section on common issues I promised earlier today too.

**Comment by kousu** [comment]
@jcohenadad you'll have to merge this. I'm not a member of this repo yet.

**Comment by jcohenadad** [comment]
hum we should probably fix CI before being able to merge anything

**Comment by jcohenadad** [comment]
@kousu should we also add doc for maintainer of this repos? e.g., i was trying to clone this repos on compute canada (to start doing some processing), and ran into:
~~~
git clone https://github.com/spine-generic/data-multi-subject.git
cd data-multi-subject
git annex init
git-annex: getUserEntryForID: does not exist (no such user)
~~~

**Comment by kousu** [comment]
> @kousu should we also add doc for maintainer of this repos?

My intention was that, yeah, this would cover that. In CONTRIBUTING.md there's a troubleshooting section. We can dump things we run into in there. I expect there to be many glitches.

**Comment by kousu** [comment]
I've added a note to the docs about checking your git-annex version.

datalad's docs on the matter just say https://www.datalad.org/get_datalad.html

> There is one additional dependency — git-annex — that is not installed automatically when using pip. Be sure to have a recent version of git-annex installed on your system.

I tried to give a bit more guidance.

**Comment by kousu** [comment]
Also these docs should probably live in a more general wiki but can they just go here for now pretty please?

**Comment by kousu** [comment]
> hum we should probably fix CI before being able to merge anything

@jcohenadad #6

File: CONTRIBUTING.md
Comment (jcohenadad) [inline CONTRIBUTING.md]:
```suggestion
git annex sync yourusername  # if you cloned this repository and you are not working on a fork, you can omit 'yourusername'
```

File: CONTRIBUTING.md
Comment (jcohenadad) [inline CONTRIBUTING.md:33]: @kousu i see these 3 PR options:

image

should it be "jca/participants"?

File: README.md
Comment (jcohenadad) [inline README.md:16]:
```suggestion
The processing pipeline to analyze this dataset is available in the [spine-generic documentation](https://spine-generic.readthedocs.io/en/latest/analysis-pipeline.html).
```

File: README.md
Comment (jcohenadad) [inline README.md]:
```suggestion
We are using a tool to manage large datasets called `git-annex`. To download, dataset, you need to have `git` installed, and also [install `git-annex`](https://git-annex.branchable.com/install/) *at version 8*. Then run:
```

Comment (jcohenadad) [inline README.md]:
```suggestion
We are using a tool to manage large datasets alled `git-annex`. To download this dataset, you need to have `git` installed, and also [install `git-annex`](https://git-annex.branchable.com/install/) *at version 8*. Then run:
```

File: README.md
Comment (jcohenadad) [inline README.md:32]: easier to type
```suggestion
get annex get sub-nwu01 sub-nwu03 sub-nwu04 sub-oxfordFmrib04 sub-tokyoSkyra*
```

File: README.md
Comment (jcohenadad) [inline README.md]: duplicate of section analysis below
```suggestion
```

File: CONTRIBUTING.md
Comment (kousu) [reply CONTRIBUTING.md:33]: Yes. `git-annex` and github aren't really designed to work together I guess!

File: README.md
Comment (kousu) [reply README.md]: :+1:

File: README.md
Comment (kousu) [reply README.md:32]: I read the `/`s as implying "folder". Also, they show up automatically if you use tab-completion.

Comment (jcohenadad) [reply README.md:32]: ^ i'm fine with that

File: CONTRIBUTING.md
Comment (kousu) [reply CONTRIBUTING.md]: Depends: are we going to be working in branches or working in forks? You told me before you wanted to protect this dataset and so wanted to force everything to be done by forks.

It could go either way but I think we should pick one and write the instructions to that. If someone wants to do it another way they can ask around and figure it out.

Comment (jcohenadad) [reply CONTRIBUTING.md]: i mostly wrote this for myself 😬  (i cloned this repos and didn't feel like forking it and re-downloading everything)

**Pull request merged**
